facility-manager.html

```diff
@@ -605,6 +605,11 @@
 
 <p>There are two ways to do this.</p>
 
+<p>
+    Do not remove users from a group where they own data - make sure the data
```

Comment:
This sentence is inserted into unlucky place in the doc. It is just after ``There are two ways to do this...``, after whcih sentence I really need to be told the two ways, not something distracting. This warning sentence here might go up or maybe a two-sentences up.